setup/fio-setup-basic

```diff
@@ -184,13 +185,18 @@ no=0
 for storage in $storages; do
 	if [ "$raw_disk" = 1 ]; then
 		storage_setup="filename=$storage"
-	elif [ "$testfile" = 'y' ]; then
-		storage_setup="\
+	else
+		dev=/dev/$(basename $storage)
+		if is_null_blk $dev; then
+			storage_setup="filename=/dev/nullb0"
+		elif [ "$testfile" = 'y' ]; then
+			storage_setup="\
```

Comment:
i propose code like below, can it work?

if [ "$raw_disk" = 1 ]; then
  storage_setup="filename=$storage"
elif is_null_blk /dev/$(basename $storage); then
  storage_setup="filename=/dev/nullb0"
elif